nodes/sros/sros.go

```diff
@@ -220,22 +221,22 @@ func (n *sros) PreDeploy(_ context.Context, params *clabnodes.PreDeployParams) e
 	// store provided pubkeys
 	n.sshPubKeys = params.SSHPubKeys
 
-	// generate the certificate
-	certificate, err := n.LoadOrGenerateCertificate(params.Cert, params.TopologyName)
-	if err != nil {
-		return err
-	}
-
-	// set the certificate data
-	n.Config().TLSCert = string(certificate.Cert)
-	n.Config().TLSKey = string(certificate.Key)
-
 	if strings.HasPrefix(n.Cfg.NetworkMode, "container:") {
 		n.Cfg.ExtraHosts = nil
 	}
 
-	// either the non-distributed OR distributed AND is a CPM
+	// Create files/dir structure for standalone nodes or distributed CPM nodes
```

Comment:
Just wondering, how many certs we generate for distributed nodes with multiple CPM cards. IMHO, we should only create one key/cert per chassis.